### DISCUSSION TITLE
New Function to Request Signature At Adoption Gone

### DISCUSSION THREAD

**Discussion by Mdeprima64** [sheltermanager/asm3#1392] [Q&A / Help] [answered]
I really liked the recently introduced function that presents the option to pick the form to create and send it to the adopter via email for signature all in one step when processing an adoption.  But it literally disappeared tonight between processing one and the next.  I signed off and back on and it's still gone.  See screenshot.

![Screenshot_20240101-033418](https://github.com/sheltermanager/asm3/assets/155357338/8eb49147-7b33-4aa5-affd-8c7f060a782c)

**Comment by Tetzor** [comment, marked as answer]
Hi,

The paperwork section wont appear until you select a person as the system needs the email address from the person record.

Kind regards,

Jon
